app/src/main/java/com/taskruler/MainViewModel.kt

```diff
@@ -77,7 +77,7 @@ class MainViewModel(var activityService : IActivityService = ActivityService())
      */
     fun getActivities() {
         viewModelScope.launch {
-            var innerActivities = activityService.getActivities()
+            val innerActivities = activityService.getActivities()
```

Comment:
changed var to val since innerActivities is never modified after initialization.